src/main/java/com/solvd/laba/Exceptions/PatientNotFoundException.java

```diff
@@ -0,0 +1,17 @@
+package com.solvd.laba.Exceptions;
+
+public class PatientNotFoundException extends Exception {

```

Comment:
Those exceptions will not count as custom. 
There is no differenve in your exceptions except for a name. Try to think how to make them more specific.